vllm/transformers_utils/configs/chameleon.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         rope_scaling=None,
         attention_bias=False,
         attention_dropout=0.0,
-        qk_layernorm=False,
```

Comment:
this is not a thing anymore?

Reply:
In the latest version they removed this config and always assume q_norm and k_norm needs to be applied